src/Components/Web.JS/tests/DefaultReconnectionHandler.test.ts

```diff
@@ -58,23 +58,23 @@ describe('DefaultReconnectionHandler', () => {
     expect(reconnect).toHaveBeenCalledTimes(1);
   });
 
-  it('invokes failed if reconnect fails', async () => {
-    const testDisplay = createTestDisplay();
-    const reconnect = jest.fn().mockRejectedValue(null);
-    const handler = new DefaultReconnectionHandler(NullLogger.instance, testDisplay, reconnect);
-    window.console.error = jest.fn();
-
-    handler.onConnectionDown({
-      maxRetries: 3,
-      retryIntervalMilliseconds: 20,
-      dialogId: 'ignored'
-    });
-
-    await delay(100);
-    expect(testDisplay.show).toHaveBeenCalled();
-    expect(testDisplay.failed).toHaveBeenCalled();
-    expect(reconnect).toHaveBeenCalledTimes(3);
-  });
+  // it('invokes failed if reconnect fails', async () => {
+  //   const testDisplay = createTestDisplay();
+  //   const reconnect = jest.fn().mockRejectedValue(null);
+  //   const handler = new DefaultReconnectionHandler(NullLogger.instance, testDisplay, reconnect);
+  //   window.console.error = jest.fn();
+
+  //   handler.onConnectionDown({
+  //     maxRetries: 3,
+  //     retryIntervalMilliseconds: 20,
+  //     dialogId: 'ignored'
+  //   });
+
+  //   await delay(500);
+  //   expect(testDisplay.show).toHaveBeenCalled();
+  //   expect(testDisplay.failed).toHaveBeenCalled();
+  //   expect(reconnect).toHaveBeenCalledTimes(3);
+  // });
```

Comment:
Don't check in commented-out code. This can be found in previous commits if needed in the future.

Reply:
This is tracked by https://github.com/aspnet/AspNetCore/issues/12578. I would rather add a comment explaining why this is commented out rather than delete the actual test code. Though I don't feel strongly. @SteveSandersonMS ?

Reply:
Commenting this out is the closest thing we have to a `[Skip]` attribute. It will be uncommented soon, so as long as there is a comment at the top explaining why this is here, and linking to the issue #12578, I think it's a fair way to express the same intent as `[Skip]`.

Reply:
Have you tried `it.skip(...)`?